src/tools/pub/versioning.md

```diff
@@ -341,8 +341,8 @@ result is a complete list of every package that your app depends on either
 directly or indirectly and the best version of that package that will work with
 your app's constraints.
 
-Pub takes that and writes it out to a **lockfile** in your app's directory
-called `pubspec.lock`. 
+Pub takes that information, computes a [content hash][] from it,
+and writes both to a [**lockfile**][] in your app's directory called `pubspec.lock`. 
```

Comment:
I don't think this link will work since it doesn't match the definition later. You should be able to pull out the bold:

```suggestion
and writes both to a **[lockfile][]**
in your app's directory called `pubspec.lock`. 
```

Reply:
Hmm, that didn't work but adding the asterisks to the definition at the bottom does.